packages/um/package.py

```diff
@@ -150,6 +152,34 @@ class Um(Package):
             "fcm_name": "netcdf",
             "fcm_ld_flags": "-lnetcdff -lnetcdf"}}
 
+    # Set directory slugs for use below
+    self.jules_resource_dir, self.um_resource_dir = "JULES", "UM"
```

Comment:
You don't need to include `self` when declaring/defining the member variables. `self` is used in member functions when referencing the instantiated object's member variables.